apps/aecore/src/aec_chain_state.erl

```diff
@@ -431,7 +431,7 @@ internal_insert(Node, Block, Origin) ->
             try aec_db:ensure_transaction(Fun)
             catch exit:{aborted, {throw, ?internal_error(What)}} -> internal_error(What)
             end;
-        {ok, Node} -> ok;
+        {ok, Node} -> {ok, #{}};
```

Comment:
On line 444 there is also an incorrect return of `ok`

Reply:
yes, but it's not relevant to return of the function

Reply:
I find the semantics weird. Why not simply return an error like `already_present` or so?